apps/OboeTester/app/src/main/java/com/mobileer/oboetester/DeviceReportActivity.java

```diff
@@ -103,8 +103,8 @@ private void reportDeviceInfo(Collection<AudioDeviceInfo> devices) {
         logClear();
         StringBuffer report = new StringBuffer();
         report.append("Device Report:\n");
-        report.append(MainActivity.getVersiontext());
-        report.append(Build.MANUFACTURER + ", " + Build.MODEL + ", " + Build.PRODUCT);
+        report.append(MainActivity.getVersiontext()).append("\n");
```

Comment:
Try to keep "tag:value" format. Maybe

    Device Report
    Oboe: Oboetester (58).....
    Build: Google, Pixel ...

Reply:
Changed to the following

```
App: OboeTester (59) v 2.3.0, Oboe v 1.6.2
Phone: Google, Pixel 4a (5G), bramble
```

Reply:
I like "App:".
In our group we tend to focus on phones. But we need OboeTester to support non-phone devices such as TV and Auto. 
Maybe "Device:".

Reply:
Done